releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,17 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.2.0 - 
+
+### New Features
+* Added support for feature flags, which allows feature flag settings to be saved to a Kubernetes ConfigMap.
+    * Added the `featureFlag.selectors` property for feature flag filtering.
+    * Added the `featureFlag.refresh` property for periodically fetching latest feature flag values and update Kubernetes ConfigMap accordingly.
+
+### Enhancements
+* Added support for setting `nodeSelector`, `affinity` and `tolerations` when use helm to install Azure App Configuration Kubernetes Provider. It's useful to control which nodes the Kubernetes Provider pod can be scheduled. [#858](https://github.com/Azure/AppConfiguration/issues/858)

```

Comment:
Added support for setting `nodeSelector`, `affinity` and `tolerations` when use helm to install Azure App Configuration Kubernetes Provider. It's useful for node assignment management